pkg/customresource/actionset.yaml

```diff
@@ -266,6 +268,10 @@ spec:
       type: string
       description: Progress of completion in percentage
       jsonPath: .status.progress.percentCompleted
+    - name: OnPhase
+      type: string
+      description: The phase that is being run, if the actionset is completed this would be last phase of action
```

Comment:
Wouldn't it be confusing if onPhase is still on last phase after completion? Should it give a final value instead?

Reply:
We can consider changing this to, let's say, completed once the actionset is completed. But I don't think that is going to be useful because we already specify at other places that the actionset is completed.
If we document it even once, anywhere, I think we would be good. But if we think having completed would be better, I can do that as well.

Reply:
If we decide to keep onPhase as the field, then we should either not set anything to the field once it's completed, or set `Completed`. Otherwise at one point we would say `onPhase` `xxx` but actionset status would be completed.